### PR TITLE
Add "grunt clean" task to clear temp files

### PR DESCRIPTION
**Pull request by azirbel**
@Addepar/ui2-developers

### CODE REVIEW

**Comment by cyril-sf** [comment]
Doesn't Ember-CLI expose something to do it?

**Comment by azirbel** [comment]
Looks like there is https://www.npmjs.com/package/ember-cli-clean but nothing else.

**Comment by azirbel** [comment]
And that one reinstalls, which is unexpected. (https://github.com/technomage/ember-cli-clean/blob/master/lib/command.js#L13) I prefer using this grunt task.

File: Gruntfile.js [hunk 1]
```diff
@@ -25,6 +26,8 @@ module.exports = function (grunt) {
       }
     },
 
+    clean: ['tmp', 'ember-dist', 'node_modules', 'bower_components'],
```
Comment (cyril-sf) [inline Gruntfile.js]: do we really want to remove all packages and not only our build files?

Comment (azirbel) [inline Gruntfile.js]: I guess this PR is mostly a convenience for me. I find myself `rm`'ing these folders a lot, so I can reinstall and rerun from scratch. I mostly use it when we update dependencies.

**Comment by cyril-sf** [comment]
LGTM

**Pull request merged**